src/prog_translator.h

```diff
@@ -62,7 +65,7 @@ class ProgTranslator {
   /// @brief translates solution from iface back into graph matches
   void FromProg();
 
-  const ProgTranslator::Context& ctx() const { return ctx_; }
+  const Context& ctx() const { return ctx_; }
```

Comment:
I have a working branch which revert those change:). I did not PR it
because I wished to understand :) I can PR it anytime!
On Tue, 6 Sep 2016 at 18:41, Anthony Scopatz <notifications@github.com>
wrote:

> In src/prog_translator.h
> <https://github.com/cyclus/cyclus/pull/1218#discussion_r77738551>:
>
> > @@ -62,7 +65,7 @@ class ProgTranslator {
> >    /// @brief translates solution from iface back into graph matches
> >    void FromProg();
> >
> > -  const ProgTranslator::Context& ctx() const { return ctx_; }
> > +  const Context& ctx() const { return ctx_; }
>
> This would also need to be changed back in all likelihood.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/cyclus/cyclus/pull/1218/files/fdbda934a1fdc12bf60dc9a6ecf551e9a5307164#r77738551>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOcZOogbITKpkofjz8qxhqduvGntEq5Aks5qnfoigaJpZM4GbPGH>
> .
>